test/cli/test_cli_functions.sh

```diff
@@ -953,3 +953,20 @@ function check_dbdump {
     # Remove diagnostics and dump files
     rm -rf /tmp/$rand_dir
 }
+
+function test_noobaa_cr_deletion() {
+    local resp
+    resp=$(kubectl -n ${NAMESPACE} delete noobaas.noobaa.io noobaa 2>&1 >/dev/null)
```

Comment:
```suggestion
    local resp=$(kubectl -n ${NAMESPACE} delete noobaas.noobaa.io noobaa 2>&1 >/dev/null)
```

Reply:
This is intentional actually.

`local` keyword itself generates an exit status which overrides the subcommand's exit status. In the test, subcommand's exit status is being used hence declaration was separated out.

Reply:
Correct, this is one of the bash pitfalls to watch for when trying to apply paradigms from more modern languages